dev/drivers/scripts/mesoscale/plots/jevs_mesoscale_grid2obs_plots.sh

```diff
@@ -1 +1,66 @@
-sample
+#PBS -S /bin/bash
+#PBS -N jevs_mesoscale_grid2obs_plots
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=10:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+export model=evs
+module reset
+export machine=WCOSS2
+
+# ECF Settings
+export RUN_ENVIR=nco
+export SENDECF=YES
+export SENDCOM=YES
+export KEEPDATA=YES
+export SENDDBN=YES
+export SENDDBN_NTC=
+export job=${PBS_JOBNAME:-jevs_mesoscale_grid2obs_plots}
+export jobid=$job.${PBS_JOBID:-$$}
+export SITE=$(cat /etc/cluster_name)
+export USE_CFP=YES
+export nproc=128
+
+# General Verification Settings
+export NET="evs"
+export STEP="plots"
+export COMPONENT="mesoscale"
+export RUN="atmos"
+export VERIF_CASE="grid2obs"
+export MODELNAME=${COMPONENT}
+
+# EVS Settings
+export HOMEevs="/lfs/h2/emc/vpppg/noscrub/$USER/EVS"
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
+export config=$HOMEevs/parm/evs_config/mesoscale/config.evs.prod.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}
+
+# Load Modules
+source $HOMEevs/versions/run.ver
+
+source /usr/share/lmod/lmod/init/sh
+module reset
+source $HOMEevs/modulefiles/$COMPONENT/${COMPONENT}_${STEP}.sh
+export HPC_OPT=/apps/ops/para/libs
+export MET_PLUS_PATH="/apps/ops/para/libs/intel/${intel_ver}/metplus/${metplus_ver}"
+export MET_PATH="/apps/ops/para/libs/intel/${intel_ver}/met/${met_ver}"
+export MET_CONFIG="${MET_PLUS_PATH}/parm/met_config"
+export PYTHONPATH=$HOMEevs/ush/$COMPONENT:$PYTHONPATH
+
+# Developer Settings
+export DATAROOT=/lfs/h2/emc/stmp/$USER/evs_test/$envir/tmp
+export COMINccpa=/lfs/h2/emc/ptmp/${USER}/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMINmrms=/lfs/h2/emc/ptmp/${USER}/$NET/$evs_ver/prep/$COMPONENT/$RUN
+export COMINspcotlk=/lfs/h2/emc/ptmp/${USER}/$NET/$evs_ver/prep/$COMPONENT/$RUN
```

Comment:
Are these 3 COMIN* variables needed? In my test runs I only chanced COMIN to point to emc.vpppg's stats. I left these 3 COMIN* variables unchanged and everything ran fine. I am wondering if they are an artifact of an inherited cam driver script.

Reply:
The three COMIN variables are not needed and they are an artifact of an earlier driver script.  I plan to remove these in a clean-up PR for meso/cam so I can test what can be removed.  However these can definitely be removed in this PR instead if that's preferred.

Reply:
Thanks @MarcelCaron-NOAA ! Yes, let's remove for the mesoscale PR.